README.md

```diff
@@ -1,27 +1,25 @@
 <!-- Update the title -->
-# Terraform Modules Template Project
+# Secrets manager public cert module
 
-<!--
-Update status and "latest release" badges:
-  1. For the status options, see https://terraform-ibm-modules.github.io/documentation/#/badge-status
-  2. Update the "latest release" badge to point to the correct module's repo. Replace "terraform-ibm-module-template" in two places.
--->
 [![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
```

Comment:
Can you update this to the graduated status?

Reply:
fixed in latest commit